### PR TITLE
Creating initial Scorecard : glyphs laid out for each state

### PR DESCRIPTION
**Pull request by katherinelazar**

### CODE REVIEW

File: frontend/src/Components/Glyphs.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+		    	.attr('x', width / 2)
+		    	.attr('text-anchor', 'middle')
+		    	.text(d['name']);
+		    vis.drawSingleGlyph(d, g);
+		});
+	}
+
+    render() {
+    	return ( <div>
+    				<div className="scorecard"></div>
```
Comment (jindaln) [inline frontend/src/Components/Glyphs.js:135]: small stylistic change just to align the tags

File: frontend/src/Views/Scorecard.js [hunk 2]
```diff
@@ -1,10 +1,152 @@
...
+
+    changeReq = (req) => {
+        this.setState({ electronic_request: !req})
+    }
+
+    changeNotary = (n) => {
+        this.setState({ notary: !n })
+    }
+
+
```
Comment (jindaln) [inline frontend/src/Views/Scorecard.js:28]: Remove extra whitespace

File: frontend/src/Components/Glyphs.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+		    .attr('class', 'practice')
+		    .attr('transform', (d, i) => {
+		    	const v = pentagonVertex(i);
+		    	return `translate(${v.x},${v.y})`;
+		    });
+
+    	practice.append('circle')
+		    .attr('r', circleRadius)
+		    .style('fill', d => state[d] === true ? availableColor : unavailableColor)
+    		.style('stroke', d => state[d] === true ? 'none' : unavailableColor);
```
Comment (jindaln) [inline frontend/src/Components/Glyphs.js:102]: consistent indentation

File: frontend/src/Components/Glyphs.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+		    .data(practicesAvailble)
+		    .enter().append('g')
+		    .attr('class', 'integration');
+
+		integration.append('path')
+		    .attr('d', d => {
+		    	const points = []
+		    	var i = 0;
+		    	for (var key in state) {
+				    if (state[key] === true) {
```
Comment (jindaln) [inline frontend/src/Components/Glyphs.js:74]: maybe reduce indentation?

File: frontend/src/Components/Glyphs.js [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+	];
+
+class Glyphs extends Component {
+
+	constructor(props) {
+        super(props);
+        this.state = {
+            states:props.states
+        };
+    }
```
Comment (jindaln) [inline frontend/src/Components/Glyphs.js:23]: good to align parentheses with constructor, same with line 25 and componentDidMount

File: frontend/src/Components/Glyphs.js [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+
+	componentDidMount() {
+        this.initVis();
+    }
+
+    initVis() {
```
Comment (jindaln) [inline frontend/src/Components/Glyphs.js:29]: would you mind adding a comment above your functions to explain what each thing does :)

**Review by jindaln — COMMENTED** [review]
In general, might be good to clean up the indentation a little and add comments for readability but this all looks great!

File: frontend/src/Components/Glyphs.js (hunk 4 above)
Comment (katherinelazar) [reply frontend/src/Components/Glyphs.js:74]: Hmm it doesn't appear to have this much indentation for me in Sublime, not sure what's causing this

**Review by jindaln — APPROVED** [review]

**Pull request merged**